thefuck/rules/pacman_invalid_option.py

```diff
@@ -0,0 +1,14 @@
+from thefuck.specific.archlinux import archlinux_env
+import re
+
+
+def match(command):
+    return "error: invalid option '-s'" in command.output
+
+
+def get_new_command(command):
+    opt = re.findall(r" -[dqrstuf]", command.script)[0]
+    return re.sub(opt, opt.upper(), command.script)
+
+
+enabled_by_default = true, pacman = archlinux_env()
```

Comment:
`enabled_by_default` is being declared as a `tuple(boo, str)`. That way, it's always evaluated to `True`. Please change it as such:
```suggestion
enabled_by_default = archlinux_env()
```